.gitignore

```diff
@@ -3,3 +3,7 @@
 /network
 docker-compose-env.ps1
 docker-compose-env.sh
+
+# alert manager generated files.
```

Comment:
I think it's not necessary
You can bind the volume to another path to avoid this

Reply:
by default it was set in root.

Reply:
in monitoring compose is set to 
```
            - ../monitoring/alertmanager:/etc/alertmanager
            - ../monitoring/alertmanager:/alertmanager
```